conf/base.py

```diff
@@ -152,6 +152,14 @@
     'TEST_REQUEST_DEFAULT_FORMAT': 'vnd.api+json',
     'DEFAULT_VERSIONING_CLASS': 'rest_framework.versioning.URLPathVersioning',
     'COERCE_DECIMAL_TO_STRING': False,
+    'DEFAULT_THROTTLE_CLASSES': (
+        "apps.throttles.OrgBurst",
+        "apps.throttles.OrgSustained",
+    ),
+    'DEFAULT_THROTTLE_RATES': {
```

Comment:
The throttle rate itself should be extracted from the user's JWT (it needs to be included there by profiles, which is your other task)

Reply:
Trying to figure out what that looks like - 

`'DEFAULT_THROTTLE_RATES': {
        'org_api_limit': 'rest_framework_simplejwt ...'   # is it inside simplejwt.token ... ?
    },
}`

Reply:
Or, to get the value from JWT if PROFILES_ENABLED is true, maybe something like this? 

`if PROFILES_ENABLED:
    REST_FRAMEWORK['DEFAULT_THROTTLE_RATES'] = (
        'rest_framework_simplejwt.[something.org_api_limit]'
    )`